src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+            <SourcesSelect
+              provider="azure"
+              selectedSourceId={sourceId}
+              setSelectedSourceId={setSourceId}
+              setFetchingError={setIsErrorFetchingSources}
+            />
+            <SourcesButton />
+            <>
+              {isErrorFetchingSources && (
+                <Alert
+                  variant={'danger'}
+                  isPlain={true}
+                  isInline={true}
+                  title={'Sources unavailable'}
+                >
+                  Sources cannot be reached, try again later or enter an account
+                  info for upload manually.
+                </Alert>
+              )}
+              {(isErrorFetchingSources || isErrorFetchingDetails) && (
+                <Alert
+                  variant={'danger'}
+                  isPlain
+                  isInline
+                  title={'AWS details unavailable'}
+                >
+                  Could not fetch Tenant id and Subscription id from Azure for
+                  given Source. Check Sources page for the source availability
+                  or select a different Source.
+                </Alert>
+              )}
+            </>
+          </>
+        )}
+        {!manual && (
+          <>
+            <Gallery hasGutter>
+              <GalleryItem>
+                <FormGroup label="Azure Tenant GUID" isRequired>
+                  <TextInput
+                    value={tenantId}
+                    type="text"
+                    readOnlyVariant="default"
+                    aria-label="Azure Tenant GUID"
+                    validated={
+                      tenantId
+                        ? ValidatedOptions.default
+                        : ValidatedOptions.error
+                    }
+                    isRequired
+                  />
+                </FormGroup>
+              </GalleryItem>
+              <GalleryItem>
+                <FormGroup label="Subscription ID" isRequired>
+                  <TextInput
+                    value={subscriptionId}
+                    type="text"
+                    readOnlyVariant="default"
+                    aria-label="Subscription ID"
+                    isRequired
+                  />
+                </FormGroup>
+              </GalleryItem>
+            </Gallery>
+            <AzureAuthButton tenantId={tenantId} />
+            <FormGroup label="Resource group" isRequired>
+              <TypeAheadSelect
+                initialSelectOptions={resourceGroups.map((value: string) => {
+                  return { value: value, children: value };
+                })}
+                selectID="resource-group-select"
+                selected={resourceGroup}
+                setSelected={setResourceGroup}
+                placeholderText="Select a resource group"
+              />
+            </FormGroup>
+          </>
+        )}
+        {manual && (
+          <>
+            <ValidatedTextField
```

Comment:
Is it possible to show the validation error after user interacted with the inputs in any way instead of from the very beginning?

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/6661244f-9bda-4026-9d76-8dfd2e70f2af)

Reply:
I could add a minimum size before showing the validation errors. WDYT?

Reply:
Like a min length for the input? That could work. We don't have any inputs anywhere that would take less than 2 chars as far as I know.